app/addons/replication/reducers.js

```diff
@@ -406,4 +463,10 @@ export const getReplicateInfo = (state) => {
   });
 };
 
+export const getPagination = (state) => state.pagination;
+export const getPageStart = (state) => 1 + (state.pagination.page - 1) * state.pagination.docsPerPage;
+export const getPageEnd = (state) => state.pagination.docsPerPage + (state.pagination.page - 1) * state.pagination.docsPerPage;
```

Comment:
Can you use this so it shows the correct range on the last page?

```suggestion
export const getPageEnd = (state) => {
  if (state.isLoading || !state.statusDocs || state.statusDocs.length === 0) {
    return false;
  }
  const pageStart = (state.pagination.page - 1) * state.pagination.docsPerPage;
  return pageStart + state.statusDocs.length;
};
```